sphinx/texinputs/sphinxhowto.cls

```diff
@@ -91,21 +90,18 @@
 % Contents.
 % For an article document class this environment is a section,
 % so no page break before it.
-\let\py@OldThebibliography=\thebibliography
-\renewcommand{\thebibliography}[1]{
+\newenvironment{sphinxthebibliography}[1]{%
   \phantomsection
-  \py@OldThebibliography{1}
-  \addcontentsline{toc}{section}{\bibname}
-}
+  \begin{thebibliography}{\detokenize{#1}}%
```

Comment:
sometimes ``#1`` can be something like ``Ref_1``, but the underscore causes a problem. With ``\detokenize`` we silence such problems. It is better than just using ``1`` which just doesn't take into account the size of the widest label.